src/libstore/globals.hh

```diff
@@ -710,12 +710,10 @@ public:
     Setting<StringSet> trustedSubstituters{
         this, {}, "trusted-substituters",
         R"(
-          A list of [URLs of Nix stores](@docroot@/command-ref/new-cli/nix3-help-stores.md#store-url-format),
-          separated by whitespace. These are
-          not used by default, but can be enabled by users of the Nix daemon
-          by specifying `--option substituters urls` on the command
-          line. Unprivileged users are only allowed to pass a subset of the
-          URLs listed in `substituters` and `trusted-substituters`.
+          A list of [URLs of Nix stores](@docroot@/command-ref/new-cli/nix3-help-stores.md#store-url-format), separated by whitespace.
+          These are not used by default, but can be enabled by users of the Nix daemon by specifying [`substituters`](#conf-substituters).
+
+          Unprivileged users are allowed to pass as `substituters` only those URLs listed in `trusted-substituters`.
```

Comment:
```suggestion
          Unprivileged users (set in [`allowed-users`](#conf-allowed-users)) can pass as `substituters` only those URLs listed in `trusted-substituters`.
```

Reply:
Unprivileged means included in `allowed-users` *and not* including in `trusted-users`